scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -381,18 +382,20 @@ def _on_wavelength_step_type_changed(self):
             self.wavelength_stacked_widget.setCurrentIndex(0)
             self.wavelength_step_label.setText(u'Step [\u00c5^-1]')
 
-    def create_data_table(self, show_periods):
+    def create_data_table(self):
         # Delete an already existing table
         if self.data_processor_table:
             self.data_processor_table.setParent(None)
 
-        self.data_processor_table = jobtreeview.JobTreeView(
-            ["Sample Scatter", "ssp", "Sample Transmission", "stp", "Sample Direct", "sdp",
-             "Can Scatter", "csp",
-             "Can Transmission", "ctp", "Can Direct", "cdp", "Output Name", "User File",
-             "Sample Thickness", "Sample Height", "Sample Width", "Sample Shape",
-             "Options"]
-            , self.cell(""), self)
+        # This dictates the order displayed on the GUI
+        column_names = ["Sample Scatter", "Sample Transmission", "Sample Direct",
```

Comment:
Could we define this list at the top of the file and lilkewise for period columns e.c.t. so we have all the hardcoded stuff in one place?